drivers/timer/stm32_lptim_timer.c

```diff
@@ -63,7 +81,7 @@ static void lptim_irq_handler(const struct device *unused)
 		 * used in the sys_clock_cycle_get_32() function.
 		 * Reading the CNT register gives a reliable value
 		 */
-		uint32_t autoreload = LL_LPTIM_GetAutoReload(LPTIM1) + 1;
+		autoreload++;
```

Comment:
This change should imply a modification of the above comment: " * Reading the CNT register gives a reliable value"